country-a-service/cda-generator/src/main/resources/templates/commons.ftl

```diff
@@ -6,11 +6,13 @@ codeSystem="${value.codeSystem}"<#rt>
 <#if value.codeSystemName??> codeSystemName="${value.codeSystemName}"</#if><#rt>
 <#if value.codeSystemVersion??> codeSystemVersion="${value.codeSystemVersion}"</#if><#rt>
 </#macro>
+
```

Comment:
Please check if this linebreak goes into the CDA.  If it does, then I think we should revert.